Automattic/pocket-casts-android
# Bookmark icon on Delete fix

## Description

This PR fixes on deleting bookmark icon without need to reload the view. 

Fixes #  #1273 

## Testing Instructions

1. Login using paid account
2. Go to Profile tab -> Files screen
3. Add a file
4. Play the file and add a few bookmarks using the Player screen bottom toolbar "Add bookmark" option
4. Return to Files screen
5. Notice that the file row has a bookmark icon
6. Tap on the file
7. On the bottom sheet that appears, tap on Bookmarks menu
8. Delete all bookmarks
9. Check the bookmark icon is deleted.

## Screenshots or Screencast 


## Checklist
 
#### I have tested any UI changes...
